src/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  ListItem as MuiListItem,
+  Divider,
+  ListItemText,
+  ListItemProps as MuiListItemProps,
+  Box,
+} from '@mui/material';
+
+export interface ListItemProps extends MuiListItemProps {
+  /**
+   * The main text of list item row
+   * @default undefined
+   * @optional
+   */
+  headerText?: string;
+  /**
+   * a sorta subheader or more text to supplement main header
+   * @optional i
```

Comment:
hmm I assume the "i" here is a typo? lol

Reply:
😭